config/foreman.hiera/common.yaml

```diff
@@ -6,6 +6,10 @@ apache::protocols:
   - 'h2'
   - 'h2c'
   - 'http/1.1'
+# Value 150 is below the httpd default and showing warnings
+# https://httpd.apache.org/docs/current/mod/mpm_common.html#maxrequestworkers
+# https://bugzilla.redhat.com/show_bug.cgi?id=2244370
+apache::mod::event::maxclients: false
```

Comment:
Relevant puppetlabs-apache line at the relevant version: https://github.com/puppetlabs/puppetlabs-apache/blob/6023b4048b11c72e84efe1a300090345e638f0d9/manifests/mod/event.pp#L51. Note it accepts a boolean.